index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="navigation">
+            <p class="white-1 roboto-f l-height-24 f-size-18">John Doe</p>
+            <img class="menu" src="hamburguer.png" alt="">
+        </nav>
+    </header>
+
+
+    <div class="wrapper">
+
+        <div class="profile">
```

Comment:
- [ ] kindly reduce the margin (spaces) between the content of the headline and the nav bar to match Figma design 🙏 

![image](https://user-images.githubusercontent.com/89282221/193343802-24e130df-40f1-4f57-8a50-8e1c30bf065e.png)